modules/widgets/simple-payments.php

```diff
@@ -0,0 +1,330 @@
+<?php
+/*
+Plugin Name: Simple Payments
+Description: Simple Payments button implemented as a widget.
+Version: 1.0
+Author: Automattic Inc.
+Author URI: http://automattic.com/
+License: GPLv2 or later
+*/
+
+function jetpack_register_widget_simple_payments() {
+	register_widget( 'Simple_Payments_Widget' );
+}
+add_action( 'widgets_init', 'jetpack_register_widget_simple_payments' );
+
+class Simple_Payments_Widget extends WP_Widget {
+	private static $dir              = null;
+	private static $url              = null;
+	private static $product_defaults = null;
+
+	function __construct() {
+		$widget = array(
+			'classname'   => 'simple-payments',
+			'description' => __( 'Add a simple payment button.', 'jetpack' ),
```

Comment:
Should this be a bit more detailed so folks get to know how the button works, what you can pay with it?